cleanlab/datalab/issue_manager/label.py

```diff
@@ -118,9 +127,38 @@ def _reset(self) -> None:
 
     def find_issues(
         self,
-        pred_probs: np.ndarray,
+        pred_probs: Optional[npt.NDArray] = None,
+        features: Optional[npt.NDArray] = None,
         **kwargs,
     ) -> None:
+        """Find label issues in the datalab.
+
+        Parameters
+        ----------
+        pred_probs :
+            The predicted probabilities for each example.
+
+        features :
+            The features for each example.
+        """
+        if pred_probs is None:
+            if features is None:
+                raise ValueError(
+                    "Either pred_probs or features must be provided to find label issues."
+                )
+            # produce out-of-sample pred_probs from features
+            knn = KNeighborsClassifier(n_neighbors=self.k)
```

Comment:
```suggestion
            knn = KNeighborsClassifier(n_neighbors=self.k + 1)
```

To adjust for omitting one datapoint later (to get out-of-sample pred_probs). Ref: https://github.com/cleanlab/cleanlab/pull/760#discussion_r1264730502